### DISCUSSION TITLE
Is pytorch lightning using MKLDNN when no GPU is available ?

### DISCUSSION THREAD

**Discussion by AlexandreDey** [Lightning-AI/pytorch-lightning#5602] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Hi, 

I was wondering if PL can use mkldnn/dnnl/onednn ? I may be mistaken, but it seems pytorch does not default to it and there is no explicit mention of it inside PL's code. If it's not, is there a way to enable it ?

Alexandre

**Comment by carmocca** [comment, marked as answer]
No, unfortunately, MKL-DNN is currently not supported

Duplicate of https://github.com/PyTorchLightning/pytorch-lightning/issues/6020
